lib/store/src/csf/testing-utils/index.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  isExportStory,
+  AnyFramework,
+  AnnotatedStoryFn,
+  StoryAnnotations,
+  ComponentAnnotations,
+  ProjectAnnotations,
+  Args,
+  StoryContext,
+} from '@storybook/csf';
+
+import { prepareStory } from '../prepareStory';
+import { normalizeStory } from '../normalizeStory';
+import { HooksContext } from '../../hooks';
+import { normalizeComponentAnnotations } from '../normalizeComponentAnnotations';
+import { getValuesFromArgTypes, normalizeProjectAnnotations } from '..';
+import type { CSFExports, TestingStoryPlayFn, TestingStory } from './types';
+
+export * from './types';
+
+if (process.env.NODE_ENV === 'test') {
+  // eslint-disable-next-line global-require
+  const { default: addons, mockChannel } = require('@storybook/addons');
+  addons.setChannel(mockChannel());
+}
+
+let GLOBAL_STORYBOOK_CONFIG = {};
+
+export function setGlobalConfig<TFramework extends AnyFramework = AnyFramework>(
+  config: ProjectAnnotations<TFramework>
+) {
+  GLOBAL_STORYBOOK_CONFIG = config;
+}
+
+export function composeStory<
+  TFramework extends AnyFramework = AnyFramework,
+  TArgs extends Args = Args
+>(
+  story: AnnotatedStoryFn<TFramework, TArgs> | StoryAnnotations<TFramework, TArgs>,
+  meta: ComponentAnnotations<TFramework, TArgs>,
```

Comment:
```suggestion
  componentAnnotations: ComponentAnnotations<TFramework, TArgs>,
```

Were you going to change these variable names?

Reply:
It was done already, just not pushed!